interm_notebook_5_classify.ipynb

```diff
@@ -0,0 +1,1175 @@
+{
```

Comment:
<p>I see there are a few "to be linked" on this page. Here are the links we typically use for "<a href="https://archive.stsci.edu/missions-and-data/tess" target="_blank">TESS mission</a>" and "<a href="https://spacetelescope.github.io/hellouniverse/intro.html" target="_blank">Hello Universe</a>".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>